wallet/addresses.go

```diff
@@ -141,7 +141,7 @@ func (m *managedAddress) p2shScript() (uint16, []byte) {
 	s := []byte{
 		0:  txscript.OP_HASH160,
 		1:  txscript.OP_DATA_20,
-		22: txscript.OP_EQUALVERIFY,
+		22: txscript.OP_EQUAL,
```

Comment:
afaict this is correct. It was producing non-standard scripts according to decodescript, and the value was different than what stdaddr will return.

Reply:
Agree, it is not `OP_EQUALVERIFY`